src/lib/third_party/src/ndpi_crc.c

```diff
@@ -0,0 +1,97 @@
+#include "ndpi_main.h"
+#include "ndpi_crc.h"
+
+static const u_int16_t crc16_ccitt_table[256] = {
```

Comment:
Do you have any sources/references for those lookup tables?

Reply:
> Do you have any sources/references for those lookup tables?

It's just a CRC for each byte 0 through 255, the polynomials are 0x1021 and 0x8408.